leankyr/Easy_Problems/Caesar_Cipher/caesarCipher.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3.5
+# -*- coding: utf-8 -*-
+
+import sys
+
+def caesarCipher(s, k):
+    crypts = ''
+    k = k%26
+    for c in s:
+        ### HOTFIX ####
+        if c.isupper() and ord(c) == 90 - k:
+            crypts += 'Z'
+            continue
+        if c.islower() and ord(c) == 122 - k:
+            crypts += 'z'
+            continue
+        ### END HOTFIX ###
+        
+        
+        
+        if ord(c) in range(ord('A'),ord('Z')+1):
```

Comment:
This is to complicated try using [deque](https://docs.python.org/3/library/collections.html#collections.deque) for your implementation